src/components/AddBook.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const AddBook = () => (
+  <section className="add-book-form">
+    <h2>ADD NEW BOOK</h2>
+    <form className="books-list">
+      <input type="text" placeholder="Book title" />
+
+      <button type="button">ADD BOOK</button>
+    </form>
+  </section>
+);
```

Comment:
- [ ] Good job so far @jash-debug, Here we need to have two inputs. one for the title and one for the author. Pleas add this for a better design layout latter.